extractor/standalone/containers/docker/docker_test.go

```diff
@@ -0,0 +1,186 @@
+				{
+					Name:    "redis",
+					Version: "sha256:a8036f14f15ead9517115576fb4462894a000620c2be556410f6c24afb8a482b",
+					Metadata: &plugin.Metadata{
+						ImageName:   "redis",
+						ImageDigest: "sha256:a8036f14f15ead9517115576fb4462894a000620c2be556410f6c24afb8a482b",
+						ID:          "3ea6adad2e94daf386e1d6c5960807b41f19da2333e8a6261065c1cb8e85ac81",
+						Ports:       []types.Port{{IP: "127.0.0.1", PrivatePort: 6379, PublicPort: 1112, Type: "tcp"}},
+					},
+				},
+			},
+		},
+		{
+			name: "valid two containers running",
+			ctrs: []types.Container{
+				{
+					ID:      "3ea6adad2e94daf386e1d6c5960807b41f19da2333e8a6261065c1cb8e85ac81",
+					Names:   []string{"redis_server"},
+					Image:   "redis",
+					ImageID: "sha256:a8036f14f15ead9517115576fb4462894a000620c2be556410f6c24afb8a482b",
+					Command: "docker-entrypoint.sh redis-server",
+					Created: 1742382918,
+					Ports:   []types.Port{{IP: "127.0.0.1", PrivatePort: 6379, PublicPort: 1112, Type: "tcp"}},
+					Labels:  map[string]string{},
+					State:   "running",
+					Status:  "Up 4 days",
+				},
+				{
+					ID:      "57eaa9ded450bde6c214dfdced8897d6bb67f1611fa6befffc7a768b5d1cbc5a",
+					Names:   []string{"postgres_server"},
+					Image:   "postgres",
+					ImageID: "sha256:e92968df83750a723114bf998e3e323dda53e4c5c3ea42b22dd6ad6e3df80ca5",
+					Command: "docker-entrypoint.sh postgres",
+					Created: 1742814376,
+					Ports:   []types.Port{{IP: "", PrivatePort: 5432, PublicPort: 0, Type: "tcp"}},
+					Labels:  map[string]string{},
+					State:   "running",
+					Status:  "Up 8 minutes",
+				},
+			},
+			wantInventory: []*extractor.Inventory{
+				{
+					Name:    "redis",
+					Version: "sha256:a8036f14f15ead9517115576fb4462894a000620c2be556410f6c24afb8a482b",
+					Metadata: &plugin.Metadata{
+						ImageName:   "redis",
+						ImageDigest: "sha256:a8036f14f15ead9517115576fb4462894a000620c2be556410f6c24afb8a482b",
+						ID:          "3ea6adad2e94daf386e1d6c5960807b41f19da2333e8a6261065c1cb8e85ac81",
+						Ports:       []types.Port{{IP: "127.0.0.1", PrivatePort: 6379, PublicPort: 1112, Type: "tcp"}},
+					},
+				},
+				{
+					Name:    "postgres",
+					Version: "sha256:e92968df83750a723114bf998e3e323dda53e4c5c3ea42b22dd6ad6e3df80ca5",
+					Metadata: &plugin.Metadata{
+						ImageName:   "postgres",
+						ImageDigest: "sha256:e92968df83750a723114bf998e3e323dda53e4c5c3ea42b22dd6ad6e3df80ca5",
+						ID:          "57eaa9ded450bde6c214dfdced8897d6bb67f1611fa6befffc7a768b5d1cbc5a",
+						Ports:       []types.Port{{IP: "", PrivatePort: 5432, PublicPort: 0, Type: "tcp"}},
+					},
+				},
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			var input *standalone.ScanInput
+
+			cli := fakeclient.New(tt.ctrs)
+			e := plugin.NewWithClient(cli)
+
+			got, _ := e.Extract(context.Background(), input)
+
+			ignoreOrder := cmpopts.SortSlices(func(a, b any) bool {
+				return fmt.Sprintf("%+v", a) < fmt.Sprintf("%+v", b)
```

Comment:
Is the Name enough to tell the inventories apart? In that case you could just compare based on the name like in https://github.com/google/osv-scalibr/blob/efb5afd2f15bd155d27c8aa8717dd5ce25e0b28c/extractor/filesystem/os/homebrew/homebrew_test.go#L108